django_celery_results/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,191 @@
+# Brazilian portuguese translation strings for django-celery-results.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as django-celery-results.
+# Eduardo Oliveira <eduardo_y05@outlook.com>, 2022.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-01-04 19:52-0300\n"
+"PO-Revision-Date: 2022-01-04 19:52-0300\n"
+"Last-Translator: Eduardo Oliveira <eduardo_y05@outlook.com>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: admin.py:40
+msgid "Parameters"
+msgstr "Parâmetros"
+
+#: admin.py:47
+msgid "Result"
+msgstr "Resultado"
+
+#: apps.py:14
+msgid "Celery Results"
+msgstr "Resultados do celery"
+
+#: models.py:28
+msgid "Task ID"
+msgstr "Id da tarefa"
+
+#: models.py:29
+msgid "Celery ID for the Task that was run"
+msgstr "Id do celery em que a tarefa foi executada"
+
+#: models.py:32
+msgid "Periodic Task Name"
+msgstr "Nome da tarefa periódica"
+
+#: models.py:33
+msgid "Name of the Periodic Task which was run"
+msgstr "Nome da tarefa periódica que foi executada"
+
+#: models.py:36
+msgid "Task Name"
+msgstr "Nome da tarefa"
+
+#: models.py:37
+msgid "Name of the Task which was run"
+msgstr "Nome da tarefa que foi executada"
+
+#: models.py:40
+msgid "Task Positional Arguments"
+msgstr "Argumentos posicionais da tarefa"
+
+#: models.py:41
+msgid "JSON representation of the positional arguments used with the task"
+msgstr "Representação JSON dos argumentos posicionais usados pela tarefa"
+
+#: models.py:45
+msgid "Task Named Arguments"
+msgstr "Argumentos nomeados da tarefa"
+
+#: models.py:46
+msgid "JSON representation of the named arguments used with the task"
+msgstr "Representação JSON dos argumentos nomeados usados pela tarefa"
+
+#: models.py:51
+msgid "Task State"
+msgstr "Status da tarefa"
+
+#: models.py:52
+msgid "Current state of the task being run"
+msgstr "Status atual da tarefa em execução"
+
+#: models.py:55
+msgid "Worker"
+msgstr "Worker"
+
+#: models.py:55
+msgid "Worker that executes the task"
+msgstr "Worker que executou a tarefa"
```

Comment:
Acredito que aqui deveria ser "Woker que executa a tarefa" ou "Worker que roda a tarefa"
Considerando que "executes" parece ser tempo presente

I think that here should be "Woker que executa a tarefa" or "Worker que roda a tarefa", seeing that "executes" is present time.